app/concerns/long_runnable.rb

```diff
@@ -65,6 +66,7 @@ def run
 
     def run!
       @thread = Thread.new do
+        Thread.current[:name] = "#{id}-#{Time.now}"
```

Comment:
Then I think this makes sense.  I don't fully understand the change, but if
it's working better for you, let's merge it.

On Thursday, November 12, 2015, Dominik Sander <notifications@github.com>
wrote:

> In app/concerns/long_runnable.rb
> <https://github.com/cantino/huginn/pull/1142#discussion_r44676222>:
>
> > @@ -65,6 +66,7 @@ def run
> >
> >      def run!
> >        @thread = Thread.new do
> > +        Thread.current[:name] = "#{id}-#{Time.now}"
>
> No, at most every 5 seconds if the worker is constantly dying.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cantino/huginn/pull/1142/files#r44676222>.
>